README.markdown

```diff
@@ -279,12 +280,12 @@ else {
 
 * There should be exactly one blank line between methods to aid in visual clarity and organization. Whitespace within methods should separate functionality, but having too many sections in a method often means you should refactor into several methods.
 
-* Colons always have no space on the left and one space on the right. Exceptions are the ternary operator `? :`, empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
+* Colons always have no space on the left and one space on the right. Exceptions are the ternary operator `? :`, dictionary type declaration `[String : CGFloat]`, empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
```

Comment:
@FWPedro I removed the wrong dictionary explanation, and specified the rule exception for the dictionary type